test/bdminterface_test.cc

```diff
@@ -0,0 +1,29 @@
+#include <gtest/gtest.h>
+//#include "cell.h"
+#include "bdm_interface.h"
+
+namespace bdm {
+
+TEST(BDMInterfaceTest, Vector3DInterface) {
+  static Vector3DInterface v3Dinterf;
+
+  EXPECT_EQ(v3Dinterf.comp[0], 0);
```

Comment:
You have mixed up the parameters for EXPECT_EQ
it is `EXPECT_EQ(expected, actual)`
otherwise test ouput for failing tests might be confusing
also for all other occurences